_posts/2015-04-03-april-newsletter.md

```diff
@@ -0,0 +1,322 @@
+---
+layout: newsletter
+title: "Newsletter #5 - TODO"
+category: newsletter
+permalink: /news/2015/april/
+---
+
+Welcome to the fifth newsletter for Neovim.
+
+## General News
+
+### Black Duck 2014 Open Source Rookie
+
+[Black Duck][blackduck-site], a company that helps other corporations in the
+adoption of open source software, chose Neovim as one of the projects to feature
+of this last year. The company annually recognizes some of the top new open
+source projects that started during the year.
+
+### ICCF Holland Donations
+
+As mentioned in the last newsletter, [Unixstickers][iccf-unixstickers] has added
+[Neovim stickers][iccf-stickers] to their inventory. Every sticker that is
+bought increases the amount donated to [ICCF Holland][iccf-site], Bram
+Moolenaar's charity.
+
+Since they've been added, USD 106 has been raised which Unixstickers [just
+donated][iccf-tweet] on March 24th.
+
+### Related Projects
+
+The [Neovim Wiki][related-main-wiki] contains a page titled [Related
+projects][related-wiki] that contains a list of current GUI projects, API
+clients, plugins that take advantage of Neovim features, and projects with
+direct support for Neovim.
+
+Some of the projects might be experimental which means there might be bugs and
+the projects might still be in progress.
+
+Do you know of another project that can be listed? Don't be afraid to add a link
+yourself!
+
+## Development News
+
+### Replacing Terminal UI
```

Comment:
can you change this to:

> ### Terminal UI (TUI)

To introduce people to the term "TUI" which is very useful for us.